nuxeo-retention/src/test/java/org/nuxeo/retention/test/RetentionFeature.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * (C) Copyright 2025 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Guillaume Renard
+ */
+
+package org.nuxeo.retention.test;
+
+import org.nuxeo.ecm.automation.test.EmbeddedAutomationServerFeature;
+import org.nuxeo.ecm.core.test.CoreSearchFeature;
+import org.nuxeo.ecm.core.test.DefaultRepositoryInit;
+import org.nuxeo.ecm.core.test.annotations.Granularity;
+import org.nuxeo.ecm.core.test.annotations.RepositoryConfig;
+import org.nuxeo.runtime.test.runner.Deploy;
+import org.nuxeo.runtime.test.runner.Features;
+import org.nuxeo.runtime.test.runner.RunnerFeature;
+
+@Features({ EmbeddedAutomationServerFeature.class, CoreSearchFeature.class })
+@RepositoryConfig(init = DefaultRepositoryInit.class, cleanup = Granularity.METHOD)
+@Deploy("org.nuxeo.ecm.platform.types")
+@Deploy("org.nuxeo.ecm.platform.webapp.types")
+@Deploy("org.nuxeo.ecm.default.config")
+@Deploy("org.nuxeo.ecm.platform.search.core")
+@Deploy("org.nuxeo.ecm.platform.tag")
+@Deploy("org.nuxeo.retention.core:OSGI-INF/retention-core-types.xml")
+@Deploy("org.nuxeo.retention.core:OSGI-INF/retention-adapters.xml")
+@Deploy("org.nuxeo.retention.core:OSGI-INF/retention-vocabularies.xml")
+@Deploy("org.nuxeo.retention.core:OSGI-INF/retention-content-template.xml")
+@Deploy("org.nuxeo.retention.core:OSGI-INF/retention-service-framework.xml")
+@Deploy("org.nuxeo.retention.core:OSGI-INF/retention-listeners.xml")
+@Deploy("org.nuxeo.retention.core:OSGI-INF/retention-operations.xml")
+@Deploy("org.nuxeo.retention.core:OSGI-INF/retention-actions.xml")
```

Comment:
Not just `@Deploy("org.nuxeo.retention.core")`?

Reply:
We did not want/need to deploy https://github.com/nuxeo/nuxeo-retention/blob/lts-2025/nuxeo-retention/src/main/resources/OSGI-INF/retention-pageproviders.xml#L32 (which leverages `ElasticSearchNxqlPageProvider`) for unit tests. That's why.

But with the work you did on the search service, I'll be able to simply `@Deploy("org.nuxeo.retention.core")` now